docs/fundamentals/code-analysis/style-rules/naming-rules.md

```diff
@@ -65,21 +65,23 @@ dotnet_naming_symbols.types.applicable_kinds = class, struct, interface, enum, d
 dotnet_naming_symbols.types.applicable_accessibilities = public, internal, private, protected, protected_internal, private_protected
 ```
 
-## Naming rule properties
+## \<propertyName> and \<propertyValue>
+
+### Naming rule properties
 
 All naming rule properties are required for a rule to take effect.
 
 | Property | Description |
 | -- | -- |
-| `symbols` | The title of a symbol group; the naming rule will be applied to the symbols in this group |
-| `style` | The title of the naming style which should be associated with this rule |
+| `symbols` | The name of a symbol group; the naming rule will be applied to the symbols in this group |
+| `style` | The name of the naming style which should be associated with this rule |
```

Comment:
"Previously defined in the file" isn't quite accurate, because AFAICT both the style and the symbol group could be defined later in the file. Would it be useful to rearrange the order: put first the symbol group properties, then the naming style properties, and finally the naming rule properties?

Reply:
Adding to the confusion: in theory you could have another editorconfig in a parent folder defining the symbol group or naming style; it wouldn't be in the same file at all. Arguably something of an edge case, but maybe just "defined elsewhere".